test/bigtest/tests/resource-show-test.js

```diff
@@ -258,6 +258,22 @@ describe('ResourceShow', () => {
       it('should display closed accordion by default', () => {
         expect(ResourcePage.usageConsolidationSection.accordion.isOpen).to.be.false;
       });
+
+      it('should display year filter', () => {
+        expect(ResourcePage.usageConsolidationSection.filters.yearDropdown.isPresent).to.be.true;
+      });
+
+      it('should display correct value for year filter', () => {
+        expect(ResourcePage.usageConsolidationSection.filters.yearDropdown.value).to.equal('2020');
```

Comment:
Please use `new Date().getFullYear()` here or we'll get a New Year's present in form of failing tests 😄

Reply:
Nice suggestion. Thanks!